### PR TITLE
Playlist

### PR DESCRIPTION
**Pull request by mimansha-swarup**
email: [adarshbalika@gmail.com](mailto:adarshbalika@gmail.com)
password: adarshBalika123

- playlist management
- default screen when data is not present

ps: Authentication need to be implemented as taught is yesterday's class

https://user-images.githubusercontent.com/69811810/162144442-4ffe5caa-7ad8-4e14-acdf-e22ab1b002b5.mp4

### CODE REVIEW

File: src/Context/index.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import { AuthProvider, useAuth } from "./authContext";
 import { ActivitiesProvider, useActivities } from "./activitiesContext";
 import { ToastProvider, useToast } from "./toastContext";
 import { useVideo, VideoProvider } from "./videosContext";
+import { PlayListProvider, usePlaylist } from "./playlistContext";
```
Comment (kiranpratyush) [inline src/Context/index.js:6]: we can write export * from "/..." instead of importing and exporting

File: src/Context/playlistContext.js [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+    (async () => {
+      try {
+        const response = await axios.get(playlistApi, {
+          headers: { authorization: authState?.token },
+        });
+        if (response.status === 200) {
+          setPlaylists(response.data.playlists);
+        }
+      } catch (error) {
+        console.log("error from use effect in playlist Context", error);
```
Comment (kiranpratyush) [inline src/Context/playlistContext.js:33]: We can write console.error()

**Review by kiranpratyush — APPROVED** [review]
Nice UI and code is consistent. Nice work. Good to go for me

File: src/Context/index.js (hunk 1 above)
Comment (mimansha-swarup) [reply src/Context/index.js:6]: are nice
thanks for telling

File: src/Components/PlaylistModal/PlaylistModal.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  const [isCreate, setIsCreate] = useState(false);
+  const {
+    playlistsState
+  } = usePlaylist();
+  
+
+
+
+
+
```
Comment (manojasarsa) [inline src/Components/PlaylistModal/PlaylistModal.js]: you can remove these blank lines

**Review by manojasarsa — APPROVED** [review]
Rest code looks clean

**Pull request merged**